<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <!--        Interactive marker to command the end effector-->
    <node name="interactive_reference" pkg="mppi_panda" type="interactive_marker.py" output="screen">
        <param name="marker_server_name" value="reference_marker_server"/>
        <param name="subscribe_initial_pose" value="true"/>
        <param name="initial_pose_topic" value="/end_effector"/>
        <param name="frame_id" value="world"/>
        <param name="target_pose_topic" value="/end_effector_pose_desired"/>
    </node>
</launch>